### ISSUE TITLE
The filter section doesn't make sense and is hard to follow

### ISSUE THREAD

**Issue by WAB0003** [learn-co-curriculum/react-hooks-state-arrays#13]
### Canvas Link

https://learning.flatironschool.com/courses/6273/pages/react-state-and-arrays?module_item_id=544489

### Concern

The step by step follow through is fine until you get to the filter part. It's unclear how the code works and, with it being a new topic, its unclear where to put stuff and remove stuff.

### Suggested Changes

Add more steps to make it clear what is being done.

**Comment by lizbur10** [comment]
Hi @WAB0003 ! Thanks for providing this feedback. We will begin updating the Phase 2 content a bit later this year, so I'm going to leave this issue open. We will address it when we make those revisions.

Thanks for contributing to the Flatiron School curriculum 💙
